src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -450,6 +450,8 @@ public async Task<HttpResponseMessage> SendAsyncCore(HttpRequestMessage request,
                 }
                 else
                 {
+                    if (NetEventSource.IsEnabled) Trace($"Request content is not null, start processing it. hasExpectContinueHeader = {hasExpectContinueHeader}");
```

Comment:
Nit: my preference would be for tracing to not include variable names, e.g. that the tracing use "Has Expect: 100-Continue header: " instead of "hasExpectContinueHeader".  Variables names can easily change, leading to tracing that actually outputs the wrong thing, plus variable names don't necessarily mean anything to customers looking at traces.